mkdocs.yml

```diff
@@ -1,9 +1,9 @@
-site_name: "Scality S3 CSI Driver"
-site_description: Documentation for Scality S3 CSI Driver.
-site_author: "Scality Engineering"
-site_url: https://scality.github.io/mountpoint-s3-csi-driver
-repo_url: https://github.com/scality/mountpoint-s3-csi-driver
-edit_uri: edit/main/docs/            # ← branch that holds Markdown
+site_name: 'Scality S3 CSI Driver'
+site_description: 'Documentation for Scality S3 CSI Driver'
+site_author: 'Scality Engineering'
+site_url: 'https://scality.github.io/mountpoint-s3-csi-driver/'
```

Comment:
Ending slash is recommended so I used this oppertunity to also add proper quotes.
Reference: https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/#usage

Reply:
Yeah! I know but just going with Mkdocs standard.